src/components/alerts/ToastContainer/ToastContainer.tsx

```diff
@@ -2,7 +2,28 @@ import React from 'react';
 import { toast as toastDefault, ToastContainer as ToastContainerDefault, ToastContainerProps } from 'react-toastify';
 import { injectStyle } from 'react-toastify/dist/inject-style';
 
-injectStyle();
+const injectToastStyles = () => {
```

Comment:
Do you need the try/catch if all the catch does is continue? Maybe just this would be fine:
```javascript
	for (const sheet of document.styleSheets) {
		const rules = sheet?.cssRules;
		const node = sheet?.ownerNode;

		if (rules?.[0].cssText.includes(':root') && rules?.[1].cssText.includes('.Toastify')) {
			node?.remove();
			document.head.insertBefore(node as Node, document.head.firstChild);

			break;
		}
	}
```

Reply:
I do, because some style sheets aren't accessible, and if you try to access `cssRules` on an inaccessible sheet, it throws an error and blows up :/